pytest_splunk_addon/helmut_lib/SearchUtil.py

```diff
@@ -162,6 +162,13 @@ def checkQueryCountIsZero(self, query, max_time=120):
             self.logger.debug("Count of results is > 0, it is:%d", result_count)
             return False, job.get_results()
 
+    def doSearch(self, query, max_time=20):
```

Comment:
- Can we update the function name to get_search_results as function returns the events for the search query.
- Also please add docstring to the function reference: https://github.com/splunk/pytest-splunk-addon/blob/main/pytest_splunk_addon/standard_lib/addon_parser/props_parser.py#L79-L87

Reply:
changes done and pushed to the repo
although I am not sure about function name change, all others class methods are following camelCase naming convention